### PR TITLE
Add social media home menu item (EXPOSUREAPP-11192)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
[Jira Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11192)

New Social Media Home Menu Item that leads the user to the social media section url in the external browser.

grafik

Known Issue that will be fixed soon:
- [x] Left / Top margin of new social media icon a bit off compared to the other icons - currently aligning with the designers

### CODE REVIEW

**Review by KathrinLuecke — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt [hunk 1]
```diff
@@ -143,6 +142,20 @@ class HomeFragment : Fragment(R.layout.home_fragment_layout), AutoInject {
         menu.findItem(R.id.test_nav_graph).isVisible = CWADebug.isDeviceForTestersBuild
     }
 
+    private fun MaterialToolbar.setupMenuItemClickListener() {
+        setOnMenuItemClickListener { menuItem ->
+            resetTransitions()
```
[the remark this answers is not in the document]
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt:147]: it is not necessary, but no harm of keeping it

**Comment by mtwalli** [comment]
I still see the icon issue , did you update it yet ?

**Comment by LukasLechnerDev** [comment]
> I still see the icon issue , did you update it yet ?

Not, yet. Still aligning with anja since the updated icon seems to be still a bit off ...

**Comment by LukasLechnerDev** [comment]
> > I still see the icon issue , did you update it yet ?
> 
> Not, yet. Still aligning with anja since the updated icon seems to be still a bit off ...

@mtwalli Icon is updated

**Review by jurajkusnier — APPROVED** [review]
LGTM

**Review by mtwalli — APPROVED** [review]

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Pull request merged**